Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
Ok so it should be on `layout.css`.

> maybe we should fix for the search view only by adding a specific class

As I was thinking. @core23 could you please do that?